package.json

```diff
@@ -77,6 +77,7 @@
     "stylelint": "^13.13.1",
     "stylelint-config-standard": "^22.0.0",
     "typescript": "~4.7.4",
-    "webpack-bundle-analyzer": "^4.8.0"
+    "webpack-bundle-analyzer": "^4.8.0",
```

Comment:
Fixing this issue does not require additional packages. Please revert changes to `package.json` and `package-lock.json`.